core/src/lib/crypto_openssl.cc

```diff
@@ -1477,13 +1477,15 @@ static int crypto_initialized = false;
  */
 int InitCrypto(void)
 {
-  int status;
+  int status = 0;
 
+#  if (OPENSSL_VERSION_NUMBER <= 0x10000000L)
```

Comment:
You cannot remove that for OpenSSL 1.0.0. Otherwise locking won't work any threading will break.
However, you can probably remove it for OpenSSL 1.1.0, but please make really sure none of the removed calls is required (we had broken that in the past already).